packages/app/src/store/local.js

```diff
@@ -0,0 +1,37 @@
+import Network from '../web3/Network'
```

Comment:
Is saving the token in local storage useful if it's already persisted as a secure token? One of the nice things of secure tokens is that it's protected from JS :)

Reply:
Not sure I follow this comment, I tried to access the SID from the cookies from JS but I couldn't, that's why I stored them in local storage

Reply:
I thought the main point of using cookies was that you cannot access it from js? SID should be set automatically by the browser when doing HTTP requests and then server decides from that header what your permissions are. You should never see SID on the client.

Reply:
Yes, it is actually doing that automatically, this is simply to know if the admin has logged in or not to show some options in the client, basically after a successful login, I set a local storage variable to identify that state